docker-compose.test.yml

```diff
@@ -12,4 +12,10 @@ services:
       - ALLOW_RESET=True
       - IS_PERSISTENT=True
     ports:
-      - 8000:8000
+      - "${CHROMA_PORT:-8000}:8000"
```

Comment:
@codetheweb I python tests don't like the default here. Lemme check.

Reply:
actually it's the test_net.

Reply:
looks like it was resolved?

Reply:
weird failure in linting. Reran, it's all fine now.